examples/base_ac/main.tf

```diff
@@ -248,7 +282,7 @@ module "ac_vm" {
   depends_on = [
     module.zpa_provisioning_key,
     local_file.user_data_file,

```

Comment:
same comment

@willguibr this file is actually not needed and we ran into a weird terraform/aws dependency problem with customers that weren't storing this file after the initial run in their CICD. you can just remove both of those lines of local_file in depends_on for all examples reference